index.html

```diff
@@ -4,15 +4,11 @@
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Speedometer 2.1</title>
     <link rel="stylesheet" href="resources/main.css">
-    <script src="resources/main.js" defer></script>
```

Comment:
Not sure why this shows up in the diff.
Once PR #26 has landed this should disappear.